README.md

```diff
@@ -591,6 +591,20 @@ Anyway::Settings.default_config_path = ->(name) { Rails.root.join("data", "confi
 
 See [environment variables](#environment-variables).
 
+3) **Doppler loader**:
+
+We also can load data from [Doppler](https://www.doppler.com/). If you want to enable this feature you must add Doppler loader.
+
+```ruby
+Anyway.loaders.append :Doppler, Anyway::Loaders::Doppler
```

Comment:
Can we automatically add the loader if the `DOPPLER_TOKEN` env var is present? (And, probably, add another en var to allow opt-out from this, say, `ANYWAY_CONFIG_DISABLE_DOPPLER=true`)

Reply:
thank you, done